Update npm to >= 2.13.1

Requested by @DFurnes 
Should fix https://jenkins.dosomething.org/job/Deploy-Staging/144/console
```
dosomething@ip-10-100-65-7:/var/www/staging.beta.dosomething.org/releases/20150723202759/lib/themes/dosomething/paraneue_dosomething$ npm install
npm ERR! 404 404 Not Found: dosomething
npm ERR! 404
npm ERR! 404 'dosomething' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
```

--- a/roles/dosomething.dosomething-dev/tasks/testing-utilities.yml
+++ b/roles/dosomething.dosomething-dev/tasks/testing-utilities.yml
@@ -7,4 +7,5 @@
     - name: mocha-phantomjs
       version: 3.*
     - name: casperjs
-      version: 1.*
+      # node 1.2 requires caspser 1.1 branch
+      version: 1.1.0-beta3

--- a/roles/dosomething.nodejs/defaults/main.yml
+++ b/roles/dosomething.nodejs/defaults/main.yml
@@ -2,5 +2,5 @@
 # defaults file for nodejs
 
 # Nodejs
-nodejs_version: "0.10"
+nodejs_version: "0.12"
 nodejs_packages_update: no

--- a/roles/dosomething.nodejs/defaults/main.yml
+++ b/roles/dosomething.nodejs/defaults/main.yml
@@ -3,4 +3,3 @@
 
 # Nodejs
 nodejs_version: "0.12"
-nodejs_packages_update: no

--- a/roles/dosomething.nodejs/meta/main.yml
+++ b/roles/dosomething.nodejs/meta/main.yml
@@ -13,4 +13,3 @@ galaxy_info:
   - system
 dependencies:
   - { role: "dosomething.base" }
-  - { role: "franklinkim.nodejs", sudo: yes }

--- a/roles/dosomething.nodejs/tasks/install.yml
+++ b/roles/dosomething.nodejs/tasks/install.yml
@@ -1,0 +1,16 @@
+---
+
+- name: Node.js | Adding APT key
+  sudo: yes
+  apt_key: >
+    url=https://deb.nodesource.com/gpgkey/nodesource.gpg.key
+    id=68576280
+
+- name: Node.js | Add Repo
+  sudo: yes
+  apt_repository: >
+    repo="deb https://deb.nodesource.com/node_{{ nodejs_version }} {{ ansible_distribution_release }} main"
+
+- name: Node.js | Install
+  sudo: yes
+  apt: name=nodejs state=latest

--- a/roles/dosomething.nodejs/tasks/main.yml
+++ b/roles/dosomething.nodejs/tasks/main.yml
@@ -1,1 +1,4 @@
 ---
+
+- include: install.yml
+  tags: [nodejs]

--- a/roles/requirements.yml
+++ b/roles/requirements.yml
@@ -1,3 +1,2 @@
 ---
-- src: franklinkim.nodejs
 - src: franklinkim.pm2

